src/components/timecard/select-timecard-period-type/SelectTimecardPeriodType.jsx

```diff
@@ -24,6 +16,17 @@ const SelectTimecardPeriodType = () => {
     shouldFocusError: false,
   });
 
+  const [timePeriods, setTimePeriods] = useState([]);
+  const getResponse = async () => {
+    const response = await getTimePeriodTypes({
```

Comment:
Try/catch here as getTimePeriodTypes throws an error

In case of error getting the data, might we want to set the list of time period types to a default option?

Reply:
Added Try/Catch.

Regarding default options, maybe we should discuss and agree a global approach which would apply across the board (e.g. should the defaults be hard-coded, or cached from first retrieval?).
If we decide, for example, to hard-code the default values, timecardService might be the best place, inside the catch block, meaning the get function will always return a defined value. Let me know your thoughts and I can add an entry for this discussion to the Tech Huddle confluence page

Reply:
Agree we need a global approach and will need to discuss further on defaults (possibly with business too)

Reply:
I've added an entry to the Tech Huddle page: https://collaboration.homeoffice.gov.uk/display/EAHW/Tech+Huddle